app/jobs/google_scraping_job.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+
+class GoogleScrapingJob < ApplicationJob
+  def perform(keyword_id, keyword)
+    scrap_from_google(keyword)
+    store_result(keyword_id)
+  end
+
+  private
+
+  def scrap_from_google(keyword)
+    uri = URI.parse("https://www.google.com/search?q=#{CGI.escape(keyword)}")
+    user_agent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) ' \
+    'Chrome/85.0.4183.121 Safari/537.36'
+
+    response = HTTParty.get(uri, { headers: { 'User-Agent': user_agent } })
+    @parse_page = Nokogiri::HTML(response)
+  end
+
+  def store_result(keyword_id)
+    ActiveRecord::Base.transaction do
```

Comment:
Do we still need the transaction 🤔

Reply:
Good catch, I think we don't need it anymore.

Reply:
Fixed in 657f8f151f6bf9c0106ba30f803bdc661660156a